app/src/test/java/com/nervousfish/nervousfish/data_objects/tap/ATapDataTest.java

```diff
@@ -0,0 +1,23 @@
+package com.nervousfish.nervousfish.data_objects.tap;
+
+import org.junit.Test;
+
+import java.sql.Timestamp;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ATapDataTest {
+
+    class tmp extends ATapData {
+    }
+
+    @Test
+    public void testTimestamp() {
+        ATapData tap = new tmp();
+        Timestamp stamp = tap.getTimestamp();
+        assertTrue(System.currentTimeMillis() >= stamp.getTime());
+        assertTrue(System.currentTimeMillis() < stamp.getTime() + 100);
```

Comment:
Because 100ms is (almost) certainly more than the time that Travis needs to run the Java code between that line and the line where stamp was created, but is precise enough to almost certainly guarantee that stamp.getTime was initialized to the current time